### PR TITLE
Update makefile.linux-mingw

### PR DESCRIPTION
**Pull request by bumbacoin**

### CODE REVIEW

**Comment by ivangabriele** [comment]
@bumbacoin This downgrade should fix the `Electra: Error initializing database environment /home/ivan/.Electra! To recover, BACKUP THAT DIRECTORY, then remove everything from it except for wallet.dat.` error, no ?

**Comment by bumbacoin** [comment]
compatibility issues between berkley db versions can cause that error,
so standardising the library version should help.

**Comment by Hyjaz** [comment]
There is already pull request #4 that fixes the initialization problem.

@ivangabriele @bumbacoin

**Comment by ivangabriele** [comment]
This is not at all related to this one @Hyjaz as you can see in the changes.

File: src/makefile.linux-mingw [hunk 1]
```diff
@@ -61,6 +62,11 @@ LIBS += -l mingwthrd -l kernel32 -l user32 -l gdi32 -l comdlg32 -l winspool -l w
 HEADERS = $(wildcard *.h)
 
 OBJS= \
+    obj/groestl.o \
+    obj/blake.o \
+    obj/skein.o \
+    obj/keccak.o \
+    obj/jh.o \
```
[the remark this answers is not in the document]
Comment (bumbacoin) [reply src/makefile.linux-mingw]: i cant recall. but if they're not in the makefile now then i guess we never changed it?

**Pull request merged**